graphql/resolvers.go

```diff
@@ -3090,14 +3093,33 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 			if err != nil {
 				return false, err
 			}
+			projectMatches := false
 			for _, projectName := range perfPlugin.Projects {
 				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+					projectMatches = true
+					break
 				}
 			}
+			if !projectMatches {
+				return false, nil
+			}
+		}
+		opts := apimodels.GetCedarPerfCountOptions{
+			BaseURL:   evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+			TaskID:    *obj.Id,
+			Execution: obj.Execution,
+		}
+
+		result, err := apimodels.CedarPerfResultsCount(ctx, opts)
+		if err != nil {
+			return false, InternalServerError.Send(ctx, fmt.Sprintf("error requesting perf data from cedar: %s", err))
+		}
+		if result.NumberOfResults == 0 {
```

Comment:
This is more of a question about the perf plugin behavior - if there's no results, does that necessarily mean that the perf plugin is disabled? It seems like the perf plugin could be enabled but it just had no results.

Reply:
This field is used to determine if we should attempt to show the perfPlugin on the UI. If it has no results it won't actually display anything and wouldn't be useful for a user to see.